tutorial/dev.ipynb

```diff
@@ -0,0 +1,285 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Developing for Intake\n",
+    "\n",
+    "Intake has been designed with a plugable structure, to allow adding new data sources and other features as easily as possible. Here we demonstrate making a new plugin to read data from the [GitHub API](https://developer.github.com/v3/), using a small and handy package that already exists, `pygithub`. We will access only one particular part of the API, as a simple example.\n",
+    "\n",
+    "The directory `intake-github` contains a full package to do the job. Lets investigave its contents by changing directory - the exact line you need next will vary depending on how you are running this notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
make this a raw cell if you can't actually run it in the notebook, or include in the markdown.

Reply:
This is meant to be executed